makepanda/makepanda.py

```diff
@@ -737,6 +737,7 @@ def parseopts(args):
         SmartPkgEnable("EIGEN",     "eigen3",    (), ("Eigen/Dense",), target_pkg = 'ALWAYS')
         SmartPkgEnable("ARTOOLKIT", "",          ("AR"), "AR/ar.h")
         SmartPkgEnable("FCOLLADA",  "",          ChooseLib(fcollada_libs, "FCOLLADA"), ("FCollada", "FCollada/FCollada.h"))
+        SmartPkgEnable("ASSIMP",    "libassimp", ("assimp"), "assimp")
```

Comment:
I believe you are correct, I have the following from my Assimp package:
/usr/lib/pkgconfig/assimp.pc